src/extension.js

```diff
@@ -0,0 +1,52 @@
+const vscode = require('vscode');
+const child_process = require('child_process');
+const util = require('util');
+
+const exec = util.promisify(child_process.exec);
+
+const extensionName = "hledger";
+
+function activate(context) {
+  const accountAutocompleteProviderSubscription =
+    vscode.languages.registerCompletionItemProvider("hledger", {
+      async provideCompletionItems(
+        document,
```

Comment:
This might be too much to ask (so feel free to say no) but how hard would it be to make this file in TypeScript? It'd help me a bunch with maintenance. I don't really know the types of these, it'd help to have them inline and checked.